src/frontend/utils/conversejs/converse-plugins/chatPlugin.ts

```diff
@@ -136,6 +136,10 @@ const addChatPlugin = (xmpp: XMPP) =>
                 break;
 
               default:
+                // When websocket is enabled, stream stanzas need to be ignored
+                if (stanza.getAttribute('xmlns') === 'urn:xmpp:sm:3') {
```

Comment:
in `converse`, a namespace is added for this https://github.com/conversejs/converse.js/blob/754f1fcb256b30784780ab4bccf07f78f76018f3/src/headless/plugins/smacks/index.js#L11
Maybe we can use it (I don't know how) to not hardcode `urn:xmpp:sm:3` ?